.github/workflows/deliver.yml

```diff
@@ -143,8 +161,6 @@ jobs:
       # https://github.com/docker/setup-qemu-action
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v3
-      - name: Checkout the repository
-        uses: actions/checkout@v4
```

Comment:
Curious if/how the `cp` command on line 177 still works?

Reply:
Good catch! This is failing. After rebasing on 3.x so that our builds complete, it failed with: https://github.com/paul121/farmOS/actions/runs/13184528286/job/36804425740

> cp: cannot stat 'docker/docker-compose.testing.*': No such file or directory